ComplianceLevels.md

```diff
@@ -0,0 +1,20 @@
+# Compliance Levels
+
+The specificiation has various levels of compliance so that partner codes can
+implement various levels of specification based on their requirements and
+desired support.  
+
+## Tier 1
+- A) Verifies the JSON input
+- B) Can obtain energy/gradient/Hessian quantities
+- C) Can perform optization, response, frequency, and spectra computations and
+     record output 
```

Comment:
Tier 1 is a high bar. Suggest a Tier 0 that A) verifies the JSON input and B) performs and records output for energy calculations

Reply:
Also, fix "optimization" spelling.